lib/wrappers/javascript/contourwall.ts

```diff
@@ -0,0 +1,95 @@
+// Implement the new function
+// Implement the "pushed_frames" field, which gets increased at every show call.
+// Remove "configure_threadpool" function
+// Implement sleep_ms field parameter for show function
+// Find numpy alternative for the pixels field. You want a high-level 3D matrix, not a low-level byte array
+
+class ContourWall {
+  private lib;
+  private cw_core_ptr: null|Deno.UnsafePointer;
+  pixels = new Uint8Array(7200); 
+
+  constructor() {
+    let libSuffix = "";
+    switch (Deno.build.os) {
+      case "windows":
+        libSuffix = "dll";
+        break;
+      case "darwin":
+        libSuffix = "dylib";
+        break;
+      case "linux":
+        libSuffix = "so";
+        break;
+      default:
+        throw new Error(`'${Deno.build.os}' is not a supported operating system`)
+    }
+
+    const contourwall_path = `./contourwall_core.${libSuffix}`;
+
+    this.lib = Deno.dlopen(
+      contourwall_path,
+      {
+        "new_with_ports": { parameters: ["pointer", "pointer", "pointer", "pointer", "pointer", "pointer", "u32"], result: "pointer" },
+        "single_new_with_port": { parameters: ["pointer", "u32"], result: "pointer" },
+        "configure_threadpool": { parameters: ["u8"], result: "bool" },
+        "show": { parameters: ["pointer"], result: "void" },
+        "update_all": { parameters: ["pointer", "pointer", "bool"], result: "void"},
+        "solid_color": { parameters: ["pointer", "u8", "u8", "u8"], result: "void"},
+        "drop": { parameters: ["pointer"], result: "void"},
+      } as const,
+    );
```

Comment:
I am missing the `new` function and `configure_threadpool` does not need to be implemented.